skypy/halo/tests/test_mass.py

```diff
@@ -59,7 +59,7 @@ def test_halo_mass_function():
     assert allclose(array_output_PS, PS_massf)
 
 
-def halo_mass_sampler():
+def test_halo_mass_sampler():
```

Comment:
Again, these fixes have nothing to do with the subhalo mass sampler so put them in a separate pull request